libass/ass.h

```diff
@@ -62,9 +62,11 @@ extern "C" {
 typedef struct ass_image {
     int w, h;                   // Bitmap width/height
     int stride;                 // Bitmap stride
-    unsigned char *bitmap;      // 1bpp stride*h alpha buffer
+    unsigned char *bitmap;      // 1-byte/pixel alpha buffer
```

Comment:
`byte-per-pixel` would be more unambiguously fool-proof (this can be parsed as “byte or pixel” and leave the reader scratching their head)